templates/ts-react-apollo-client/README.md

```diff
@@ -1,26 +1,10 @@
 # Graphback React Sample application
 
-Starter React template using one of the Graphback backend templates
+A starter template for React while using one of the Graphback backend templates.
 
+## Quickstart Guide
```

Comment:
Add note:

Client application have been made for sample model. When making any changes on server client side app will change.